Source/Core/Core/HW/EXI/EXI_DeviceEthernetTCP.h

```diff
@@ -0,0 +1,38 @@
+// Copyright 2008 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <thread>
+
+#include "Common/Flag.h"
+#include "EXI_DeviceEthernetBase.h"
```

Comment:
All project includes should be relative to the source tree.

```suggestion
#include "Core/HW/EXI/EXI_DeviceEthernetBase.h"
```